api/pages/sm_mt.js

```diff
@@ -0,0 +1,156 @@
+        var truck_transmission = '';
+        if ($('#truckTransmission').val() == "A") {
+            truck_transmission = 'Automatic';
+        }
+        else truck_transmission = 'Manual';
+
+        //alert("New Truck:\n" + "num: " + truckNumber + "\nvin: " + truckVIN + "\nplate: " + truckPlateNumber + "\nmax coops: " + truckMaxCoops + "\ntransmission: " + truck_transmission);
+        //ajax code - Connor A
+        $.ajax({
+        type: "POST",
+        url: "../truck/addTruck.php", //php file goes here
+        data:{
+            truckNumber : truckNumber,
+            truckVIN : truckVIN,
+            truckPlateNumber : truckPlateNumber,
+            truckMaxCoops : truckMaxCoops,
+            customerZip : customerZip
+        },
+        success: function(response){
+            alert(response);
+        }
+    });
+	alert("got passed ajax");
+		
+
+    }
+          function addDriver(){
+        var truckNumber = $('#driverFirst').val();
+        var truckVIN = $('#driverLast').val();
+        var truckPlateNumber = $('#driver').val();
+        var truckMaxCoops = $('#truckMaxCoops').val();
+        var customerZip = $('#customerZip').val();
+        var truck_transmission = '';
+        if ($('#truckTransmission').val() == "A") {
+            truck_transmission = 'Automatic';
+        }
+        else truck_transmission = 'Manual';
+
+        //alert("New Truck:\n" + "num: " + truckNumber + "\nvin: " + truckVIN + "\nplate: " + truckPlateNumber + "\nmax coops: " + truckMaxCoops + "\ntransmission: " + truck_transmission);
+        //ajax code - Connor A
+        $.ajax({
+        type: "POST",
+        url: "../truck/addTruck.php", //php file goes here
+        data:{
+            truckNumber : truckNumber,
+            truckVIN : truckVIN,
+            truckPlateNumber : truckPlateNumber,
+            truckMaxCoops : truckMaxCoops,
+            customerZip : customerZip
+        },
+        success: function(response){
+            alert(response);
+        }
+    });
+	alert("got passed ajax");
+		
+
+    }
+    
+        function addCustomer(){
+        var customerName = $('#customerName').val();
+        var customerAddress = $('#customerAddress').val();
+        var customerPhone = $('#customerPhone').val();
+        var customerZip = $('#customerZip').val();
+        var customerCity = $('#customerCity').val();
+        //alert("New Customer:\n" + "name: " + customerName + "\naddr: " + customerAddress + "\nphone: " + customerPhone);
+        //ajax code -Connor A.
+        $.ajax({
+        type: "post",
+        url: "../store/addStore.php", //php file goes here
+        data:{
+            customerName : customerName,
+            customerAddress : customerAddress,
+            customerPhone : customerPhone,
+            customerZip : customerZip,
+            customerCity : customerCity
+        },
+        success: function(response){
+            alert(response);
+        }
+        
+    });
+        
+        }
+  

```

Comment:
keep the source of script.js in on the pages. once you do that, you'll have to delete these logout and loggedin functions so it knows which script you're referencing

using this file only won't work because it's missing the redirect and permissions functions